catalogue/webapp/pages/works.js

```diff
@@ -242,49 +230,52 @@ export const Works = ({
-                          query={query}
-                          page={page}
-                          workType={workType}
-                          itemsLocationsLocationType={
-                            itemsLocationsLocationType
-                          }
-                        />
-                      </div>
-                    ))}
-                  {!useImagelessResult &&
-                    works.results.map(result => (
-                      <div
-                        key={result.id}
-                        className={grid({ s: 6, m: 4, l: 3, xl: 2 })}
-                      >
-                        <WorkPromo
-                          id={result.id}
-                          image={{
-                            contentUrl: result.thumbnail
-                              ? result.thumbnail.url
-                              : 'https://via.placeholder.com/1600x900?text=%20',
-                            width: 300,
-                            height: 300,
-                            alt: '',
-                          }}
-                          datePublished={
-                            result.createdDate && result.createdDate.label
-                          }
-                          title={result.title}
-                          link={workUrl({ id: result.id, query, page })}
-                        />
-                      </div>
-                    ))}
+                  <TogglesContext.Consumer>
+                    {({ genericWorkCard }) => {
+                      return genericWorkCard
+                        ? works.results.map(result => (
+                            <div
+                              className={classNames({
+                                [grid({ s: 12, m: 10, l: 8, xl: 8 })]: true,
+                              })}
+                              key={result.id}
+                            >
+                              <WorkCard
+                                work={result}
+                                query={query}
+                                page={page}
+                                workType={workType}
+                                itemsLocationsLocationType={
+                                  itemsLocationsLocationType
+                                }
+                              />
+                            </div>
+                          ))
+                        : works.results.map(result => (
+                            <div
+                              key={result.id}
+                              className={grid({ s: 6, m: 4, l: 3, xl: 2 })}
+                            >
+                              <WorkPromo
+                                id={result.id}
+                                image={{
+                                  contentUrl: result.thumbnail
+                                    ? result.thumbnail.url
+                                    : 'https://via.placeholder.com/1600x900?text=%20',
```

Comment:
We should probably stop relying on this service for a grey box?

Reply:
Hopefully the WorkPromo will be usurped by the card, which allows for no image, so 💯 agreed.